### ISSUE TITLE
Migration files not copied by install generator

### ISSUE THREAD

**Issue by Spone** [Sorcery/sorcery#119]
Hello,

I'm using Sorcery 0.11.0

I'm running the install generator in my Rails 5.2.0.rc2 project, and the migration files are not copied:

```
$ rails generate sorcery:install core user_activation magic_login                                      
      create  config/initializers/sorcery.rb
        gsub  config/initializers/sorcery.rb
    generate  model User --skip-migration
      invoke  active_record
      create    app/models/user.rb
      invoke    test_unit
      create      test/models/user_test.rb
      invoke      factory_bot
      create        test/factories/users.rb
      insert  app/models/user.rb
      insert  app/models/user.rb
```

Do you know what's happening?

**Comment by prsimp** [comment]
I ran into the same issue on a fresh Rails 5.1.6 API-only application. A little digging indicates that [this line](https://github.com/Sorcery/sorcery/blob/341c7ea620605221eccd0af6b10d41a36bb9ef50/lib/generators/sorcery/install_generator.rb#L64) is the culprit. The `Sorcery::Generators::InstallGenerator::ActiveRecord` constant is not defined so the `copy_migration_files` method returns early. This line hasn't changed in 6 years, so I suspect it might be on the Rails side.

I'm just going to manually create the migrations using the templates [here](https://github.com/Sorcery/sorcery/tree/341c7ea620605221eccd0af6b10d41a36bb9ef50/lib/generators/sorcery/templates/migration). I'll add more information if I do anymore digging.

**Comment by mladenilic** [comment]
On Rails v5.2.0, migration is not copied as well.

**Comment by shiroemons** [comment]
Ruby 2.5.0 is working.
```
defined?(Sorcery::Generators::InstallGenerator::ActiveRecord)
# -> "constant"
```
Ruby 2.5.1 is not working.
```
defined?(Sorcery::Generators::InstallGenerator::ActiveRecord)
# -> nil
```

**Comment by shiroemons** [comment]
[Remove top-level constant lookup](https://bugs.ruby-lang.org/issues/11547) in Ruby 2.5.0.
However, the ```defined?``` method has not been modified.
This seems to have been modified to return nil in Ruby 2.5.1.

ref:
https://bugs.ruby-lang.org/issues/14407
https://github.com/ruby/ruby/pull/1800

**Comment by masoo** [comment]
It worked on the revision with Ruby 2.5.1.

https://github.com/masoo/sorcery/commit/547f1ad3c381043bdd25880ee59d94403a1c099a

But I think this is an incorrect fix.

**Comment by luodaoyi** [comment]
+1 with same pro problem

**Comment by Ch4s3** [comment]
This should be fixed by #128 and/or #129. Feel free to reopen if this isn't fixed on master.
